src/components/HeaderGap/index.desktop.js

```diff
@@ -2,21 +2,33 @@ import React, {PureComponent} from 'react';
 import {View} from 'react-native';
 import PropTypes from 'prop-types';
 import styles from '../../styles/styles';
+import stylePropTypes from '../../styles/stylePropTypes';
 
 const propTypes = {
     /** Styles to apply to the HeaderGap */
-    // eslint-disable-next-line react/forbid-prop-types
-    styles: PropTypes.arrayOf(PropTypes.object),
+    styles: stylePropTypes,
+
+    /** If the HeaderGap is placed in the LHP */
```

Comment:
We're more familiar with LHN than LHP.

Is this correct?
<img width="371" alt="Screenshot 2023-10-25 at 6 19 37 PM" src="https://github.com/Expensify/App/assets/108292595/b12bb297-0a7d-4d4c-870f-5b35eaa630d9">

Reply:
FYI the terminology we have decided on now has changed again 😅 

<img width="472" alt="2023-10-10_18-03-19" src="https://github.com/Expensify/App/assets/42391420/9d240c34-71a8-4ff2-9a02-d1663ee075ab">